controllers/cluster_controller_test_test.go

```diff
@@ -88,6 +95,33 @@ func TestClusterReconcilerSetupWithManager(t *testing.T) {
 	g.Expect(r.SetupWithManager(env.Manager())).To(Succeed())
 }
 
+func TestClusterReconcilerManagementClusterNotFound(t *testing.T) {
```

Comment:
can we have a test that asserts that when the management cluster exists, we set the bundle ref?

Reply:
gentle bump :)